src/components/CustomerContext.js

```diff
@@ -33,7 +33,19 @@ class CustomerProvider extends Component {
 
   componentDidMount() {
     this.setState(loadItemsFromLocalStorage());
-    this.getCustomers();
+    this.init();
+    // this.getCustomers();
+  }
+
+  init = async () => {
+    try {
+      const { session } = await fetch(`${process.env.REACT_APP_API_URL}/init`, {
```

Comment:
so this explicitly calls `/init`
and ofc you need to uncomment `this.getCusomers()`

Reply:
you'll need to change every `fetch()` through-out the React app to use this pattern:

```
fetch(`${process.env.REACT_APP_API_URL}/your-endpoint`, .....)
```